garnett/translatedstr.py

```diff
@@ -0,0 +1,82 @@
+from typing import Tuple, Optional, Callable
+from django.utils.translation import gettext as _
+
+from langcodes import Language
+from garnett.utils import get_current_language, get_current_language_code, get_languages
+
+
+class HTMLTranslationMixin:
+    def __html__(self) -> str:
+        # Add leading [lang] wrapped in a span
+        text = self
+        if not self.is_fallback:
+            return text
+        elif not self.fallback_language:
+            return "??"

```

Comment:
Can we pull out the "I don't know" response to a constant with a meaningful variable name

Reply:
Part of the demo, not needed.